scripts/install_ubuntu_packages.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+
+sudo apt-get update
+
+# Install dependencies of the cpp-driver even if they are already on CI so that we can run this locally
+sudo apt-get install -y libuv1 libuv1-dev cmake g++ libssl-dev zlib1g-dev
+
+# set VERSION to one of the tags here: https://github.com/datastax/cpp-driver/tags
```

Comment:
Need a higher-level comment: this is the version of the CPP driver that we will test this crate against.